packages/utils/interactive-hooks/src/useConst.ts

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+
+/**
+ * Hook to initialize and return a constant value. Unlike `React.useMemo`, this is guaranteed to
```

Comment:
I'm slightly confused why this needs to be a hook rather than `const var = <constant>`, but I know this is coming from FluentUI React.

Reply:
If you do something like `const address = { city: 'redmond', state: 'wa' }` the identity of that object will change (it will effectively be a different object stored in `address`) every time that line gets executed. 

`useConst` ensures that the object doesn't change.

Reply:
Ah, that makes sense, thanks!